pages/common/tlmgr-check.md

```diff
@@ -0,0 +1,28 @@
+# tlmgr check
+
+> Check the consistency of a TeX Live installation.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- Check the consistency of the whole TeX Live installation:
+
+`tlmgr check all`
+
+- Check the consistency of the whole TeX Live information in verbose mode:
+
+`tlmgr check all -v`
+
+- Check for missing dependencies:
+
+`tlmgr check depends`
+
+- Check if all TeX Live executables are present:
+
+`tlmgr check executes`
+
+- Check if all files listed in the local TLPDB are present:
+
+`tlmgr check files`
+
+- Check for duplicate filenames in the runfiles sections:
```

Comment:
```suggestion
- Check for duplicate filenames in the runfiles section:
```

Reply:
Does this need to be plural?

If not then merge (I've approved)

Reply:
That's what the man page says, so I'll just rely on that